ramlfications/utils.py

```diff
@@ -54,24 +56,39 @@ def load_schema(data):
     return data
 
 
-def requests_download_xml():
+def requests_download(url):
     try:
-        response = requests.get(IANA_URL)
+        response = requests.get(url)
         return response.text
     except requests.exceptions.RequestException as e:
         msg = "Error downloading XML from IANA: {0}".format(e)
         raise MediaTypeError(msg)
 
 
-def urllib_download_xml():
+def urllib_download(url):
     try:
-        response = urllib.urlopen(IANA_URL)
-    except urllib.URLError as e:
+        response = urllib.urlopen(url)
+    except urllib_error.URLError as e:
         msg = "Error downloading XML from IANA: {0}".format(e)
```

Comment:
With this and the `requests_download` error handling, it will reflect the incorrect message.